app/components/app_patient_nhs_discrepancy_table_component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AppPatientNHSDiscrepancyTableComponent < ViewComponent::Base
```

Comment:
```suggestion
class AppPatientNHSNumberDiscrepancyTableComponent < ViewComponent::Base
```

I'm just wondering about the naming, is the plan eventually that this component will handle all different kinds of discrepancies, or only NHS number ones?